core/src/packets/icmp/v4/mod.rs

```diff
@@ -40,18 +42,18 @@ use std::ptr::NonNull;
 /// +                         Message Body                          +
 /// |                                                               |
 /// ```
-/// The type field indicates the type of the message.  Its value
-/// determines the format of the remaining data.
+/// - *Type*:        Indicates the type of the message. Its value
+///                  determines the format of the remaining data.
 ///
-/// The code field depends on the message type.  It is used to create an
-/// additional level of message granularity.
+/// - *Code*:        This field depends on the message type. It is used to
+///                  create an additional level of message granularity.
 ///
-/// The checksum field is used to detect data corruption in the ICMPv4
-/// message and parts of the IPv4 header.
+/// *Checksum*:      This field is used to detect data corruption in the ICMPv4
```

Comment:
`Checksum` and `Message Body` missing bullets.

Reply:
fixed in v6 icmp too.